packages/@okta/vuepress-site/docs/guides/dpop/main/oktaresourceserver/requestresource.md

```diff
@@ -0,0 +1,47 @@
+## Make a request to an Okta resource
+
+<ApiLifecycle access="ea" />
+
+Access to an Okta resource requires more steps.
+
+1. Hash and base64url-encode the DPoP-bound access token for use as the `ath` value.
+1. Use the [Create the JSON Web Token](#create-the-json-web-token) section to create a DPoP proof JWT with the following claims:
+
+    Include the following required claims in the JWT payload:
+
+    * `ath`: Base64-encoded SHA-256 hash [SHS] of the DPoP-bound access token
+    * `htm`: HTTP method. The HTTP method of the request that the JWT is attached to. This value is the appropriate HTTP verb for the request. For example: `GET`.
+    * `htu`: HTTP URI. The endpoint URL for the resource that you want to access. For example: `http://${yourOktaDomain}/api/v1/${api_endpoint}`.
+    * `iat`: Issued at. Identifies the time at which the JWT is issued. The time appears in seconds since the Unix epoch. The Unix epoch is the number of seconds that have elapsed since January 1, 1970 at midnight UTC.
```

Comment:
```suggestion
    * `iat`: Issued at. The time at which the JWT is issued. The time appears in seconds since the Unix epoch. The Unix epoch is the number of seconds that have elapsed since January 1, 1970 at midnight UTC.
```

Reply:
Same content as earlier in another md.

Reply:
yes, this parameter is used in many steps. thanks.

Reply:
Yes it is. This parameter is used in multiple calls. I didn't think that I could do a stack selector within a stack selector, but I just tried and you can. So, I created a snippet for this content since it's repeated three times.